lib/target_web/controllers/api/v1/confirmation_controller.ex

```diff
@@ -0,0 +1,19 @@
+defmodule TargetWeb.API.V1.ConfirmationController do
+  @moduledoc false
+
+  use TargetWeb, :controller
+
+  alias PowEmailConfirmation.Plug
```

Comment:
```suggestion
  alias PowEmailConfirmation.Plug, as: EmailConfirmationPlug
```

`Plug` is a generic name, if the controller grows you'll lose the sense of what you are doing